src/server/system_services/account_server.js

```diff
@@ -671,9 +671,22 @@ function list_accounts(req) {
         throw new RpcError('UNAUTHORIZED', 'Must be system admin or operator');
     }
 
-    // for support account - list all accounts
+    console.log(`NBNB req.rpc_params.filter 
+        filter ${req.rpc_params.filter}, 
+        fs_identify ${req.rpc_params.filter && req.rpc_params.filter.fs_identity}, 
+        uid ${req.rpc_params.filter && req.rpc_params.filter.fs_identity && req.rpc_params.filter.fs_identity.uid}`);
     const accounts = system_store.data.accounts
+        // for support account - list all accounts
         .filter(account => is_support || !account.is_support)
+        // filter list, UID and GID together
+        .filter(account => !req.rpc_params.filter ||
+            (req.rpc_params.filter &&
+            req.rpc_params.filter.fs_identity &&
+            req.rpc_params.filter.fs_identity.uid &&
```

Comment:
you can change uid and gid to be required in the fs_identity object (in account_api ) instead of testing it here